packages/mcdoc/src/runtime/checker/index.ts

```diff
@@ -691,9 +708,15 @@ function condenseErrorsAndFilterSiblings<T>(
 	return { definitions: validDefinitions.map(d => d.definition), condensedErrors: errors }
 }
 
+interface ValidDefintionResultChildDefintion {
```

Comment:
This is a mouthful :D

Reply:
I renamed it, so it dosen't have `definition` twice anymore at least.